child_compassion/view/child_on_internet_wizard.xml

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="utf-8"?>
+<openerp>
+	<data>
+        <record id="child_on_internet_wizard_view" model="ir.ui.view">
+            <field name="name">compassion.child.on.internet.wizard</field>
+			<field name="model">child.on.internet.wizard</field>
+            <field name="type">form</field>
+			<field name="arch" type="xml">
+                <form string="Put child on internet" version="7.0">
+                    <footer>

```

Comment:
Please add a small paragraph explaining the procedure (<p>description</p>).
Explain this will put available children on the website, and add a sentence recalling the user to go update the index on typo3 after new children are sent.